plugins/scaffolder-field-extensions/scaffolder-frontend-module-http-request-field/README.md

```diff
@@ -79,3 +79,123 @@ The example template would result in the following dropdown:
 
 ![Alt text](images/dropdown_sample_closed.png?raw=true 'Example of the custom scaffolder field')
 ![Alt text](images/dropdown_sample_opened.png?raw=true 'Example of the custom scaffolder field')
+
+### Dynamic Select
+
+You can configure your http select to topggle between different requests in order to populate the dropdown depending on previously selected parameters.
+
+There are several ways to do this. The simplest looks like so:
+
+```yaml
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: custom-field-demo
+  title: Custom Field Demo
+  description: Custom Field Demo.
+spec:
+  owner: backstage/techdocs-core
+  type: service
+
+  parameters:
+    properties:
+      facetValue:
+        type: string
+        enum: ['kind', 'apiVersion']
+      custom:
+        type: string
+        # Use `SelectFieldFromApi` to configure the select field for the entry.
+        ui:field: SelectFieldFromApi
+
+        ui:options:
+          title: My Dropdown title
+          description: My custom description for the component
+          # The Path on the Backstage API and the parameters to fetch the data for the dropdown
+          path: 'catalog/entity-facets'
+          # This selects the array element from the API fetch response. It finds the array with the name kind
+          # under the facets object
+          arraySelector: 'facets.kind'
+          # (Optional) This selects the field in the array to use for the value of each select item. If its not specified
+          # it will use the value of the item directly.
+          valueSelector: 'count'
+          # (Optional) This selects the field in the array to use for the label of each select item.
+          labelSelector: 'value'
+          # (Optional)
+          dynamicParams:
```

Comment:
I wonder is it possible to use the existing params and allow substitution like this. If we wanted to go with this option maybe the `$` is a bad choice.. because it might be confused with the substitution that exists in the template steps.

```
params:
  facet: '${{parameters.facetValue}}'
  filter:
    kind: 'Component'
```

Reply:
I suppose you could also use the same substitution for path, value selector, label selector, array selector etc

Reply:
I tried that but it doesn't seem to work

Reply:
Its probably not as simple as this:

```javascript
const options = {
  // this mustache template would be passed in via the scaffolder template options
  path: "proxy/my-api/{{params.path}}"
}


const render(input: unknown, data) {
  if (typeof input === 'object') {
    return Object.keys(input).map(key => [key, renderRecursive(input[key])]);
  }
  if (typeof input === 'string') {
    return Mustache.render(input, data);
  }
  return input
}

const path = render(options.path, { paramaters: formData });
const params = renderRecursive(options.params, { paramaters: formData });
// and so on.
```

Reply:
Sorry but I don't understand what this is - where would this go?

Reply:
In the end I would be able to do something like this I mean:

```
  parameters:
    properties:
      - title: Custom
        properties:
          facet:
            type: string
          path:
            type: string
          valueSelector:
            type: string
          labelSelector:
            type: string
          custom:
            type: string
            ui:field: SelectFieldFromApi
            ui:options:
              title: My Dropdown title
              description: My custom description for the component
              path: '{{ parameters.path }}'
              params:
                facet: '{{ parameters.facet }}'
              arraySelector: 'facets.{{ parameters.facet }}'
              valueSelector: '{{ parameters.valueSelector }}'
              labelSelector: '{{ parameters.labelSelector }}'
```

Reply:
Do you want to try out your approach and see if it works? I'm not sure I get where the code you wrote should go

Reply:
https://github.com/RoadieHQ/roadie-backstage-plugins/pull/1115

Reply:
Here is a demo of that https://www.loom.com/share/6f0dd471728d4cefa23f74509945c2a5